recipes/libm2k/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+CONDA_BUILD_SYSROOT:
+  - /opt/MacOSX10.11.sdk        # [osx]
```

Comment:
Build output indicates that it's using the default 10.9 SDK, so this can probably just be removed along with the run_constrained.

Reply:
Let's try to remove those and see if it passes to avoid adding extra constraints.